### DISCUSSION TITLE
Please tell me your favorite food!

### DISCUSSION THREAD

**Discussion by naistclass** [NAIST-SE/AttractingContributors#209] [General]
https://github.com/naistclass/AttractingContributors-FavoriteFood

**Comment by TakumiHiraide** [comment]
Added . help me
https://github.com/TakumiHiraide/My-favorite-movie

**Reply by naistclass** [reply]
Thank you!
I also added.

**Comment by veekay7** [comment]
Added. Please help.
https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Reply by naistclass** [reply]
Thank you!
I also added.

**Comment by nekorobuNeko** [comment]
add!!
help me
https://github.com/nekorobuNeko/AttractingContributors-favorite-foods

**Comment by hernessteve** [comment]
Broccoli cheddar soup from panera bread. For some reason I can't check my gift card at the moment if there is still some balance left - [https://giftcards-market.com/downloads/panera-bread-gift-card/ ](https://giftcards-market.com/downloads/panera-bread-gift-card/) . The pastries and breads there are also amazing. We love to order their healthy lunches to our office, there is not even one person who was disappointed with the food from panera.

**Comment by olehvgrn** [comment]
For me, it's pizza. I often went to cafes on weekends with friends. It was wonderful. But when I read [doordash reviews](https://doordash.pissedconsumer.com/review.html), I realized that it is more profitable to order delivery and have fun at home. We often play board games and watch movies. Therefore, in order not to go for food, delivery is the best solution for us
